flexget/plugins/input/gazelle.py

```diff
@@ -0,0 +1,498 @@
+        })
+        return schema
+
+    def get_entries(self, search_results):
+        """Generator that yields Entry objects from search results"""
+        for result in search_results:
+            # Get basic information on the release
+            info = dict((k, result[k]) for k in ('artist', 'groupName', 'groupYear'))
+
+            # Releases can have multiple download options
+            for tor in result['torrents']:
+                temp = info.copy()
+                temp.update(dict((k, tor[k]) for k in ('media', 'encoding', 'format', 'torrentId')))
+
+                yield Entry(
+                    title="{artist} - {groupName} - {groupYear} "
+                          "({media} - {format} - {encoding})-{torrentId}.torrent".format(**temp),
+                    url="{}/torrents.php?action=download&id={}&authkey={}&torrent_pass={}"
+                        "".format(self.base_url, temp['torrentId'], self.authkey, self.passkey),
+                    torrent_seeds=tor['seeders'],
+                    torrent_leeches=tor['leechers'],
+                    # Size is returned in bytes, convert to MB for compat with the content_size plugin
+                    content_size=tor['size'] / (1024 ** 2)
+                )
+
+
+class InputRedacted(InputGazelleMusic):
+    """A plugin that searches RED"""
+
+    def __init__(self):
+        """Set up custom base_url and parameters"""
+        super(InputRedacted, self).__init__()
+        self.base_url = "https://redacted.ch"
+
+        self.params['encoding'].remove("q8.x (VBR)")
+        self.params['format'].remove("Ogg Vorbis")
+        self.params['media'].append("Blu-ray")
+        self.params['releasetype'].update({
+            "demo": 17,
+            "concert recording": 18,
+            "dj mix": 19,
+        })
+
+
+class InputNotWhat(InputGazelleMusic):
+    """A plugin that searches NWCD"""
+
+    def __init__(self):
+        """Set up custom base_url and parameters"""
+        super(InputNotWhat, self).__init__()
+        self.base_url = "https://notwhat.cd"
+
+        self.params['releasetype'].update({
+            'demo': 22,
+            'dj mix': 23,
+            'concert recording': 24
+        })
+        self.params['media'].extend(['Blu-ray', 'Unknown'])
+        self.params['haslog'].update({
+            "gold": 102,
+            "silver": 101,
+            "gold/silver": 100,
+            "lineage": -5,
+            "unscored": -1,
+            "missing lineage": -6,
+            "missing dr score": -7,
+            "missing sample rate": -8,
+            "missing description": -9
+        })
+
+
+@event('plugin.register')
+def register_plugin():
+    plugin.register(InputGazelle, 'gazelle', interfaces=['task', 'search'], api_ver=2)
+    plugin.register(InputGazelleMusic, 'gazellemusic', interfaces=['task', 'search'], api_ver=2)
+    plugin.register(InputRedacted, 'redactedch', interfaces=['task', 'search'], api_ver=2)
```

Comment:
We don't typically include the TLD in the plugin name unless it's part of the site name... It should just be `redacted`